src/app/models/form.server.model.ts

```diff
@@ -420,13 +423,18 @@ const compileFormModel = (db: Mongoose): IFormModel => {
   }
 
   // Transfer ownership of the form to another user
-  FormSchema.methods.transferOwner = async function (currentOwner, newOwner) {
+  FormSchema.methods.transferOwner = async function (
```

Comment:
they added a new `this` argument to instance methods?

Reply:
The typing is slightly different from the old DefinitelyTyped/mongoose types in that FormSchema.methods is now not typed at all, so it is up to us to type the methods ourselves. The old typings injected `this` automatically. Native typings will probably update the typings to be closer to DefinitelyTyped's ones sooner or later